### PR TITLE
✨ [bento][amp-date-countdown] Extend validator to 1.0 for amp-date-countdown

### PR DESCRIPTION
**Pull request by krdwan**
Similar question to the one noted in: https://github.com/ampproject/amphtml/pull/33464#pullrequestreview-621176326

The `1.0` component has a slightly different API than `0.1`.  The `1.0` component allows for an attribute `count-up` whereas the `0.1` component allows for `data-count-up`.  In the validator, it appears that both are allowed for both versions.  Is it possible to specify only `count-up` attribute for pages using the `1.0` script tag and `data-count-up` for pages using the `0.1` script tag? @ampproject/wg-caching  (context on the different API: https://github.com/ampproject/amphtml/pull/32201)

answer: no https://github.com/ampproject/amphtml/pull/33480#issuecomment-807017930

cc: @nainar 

Tracking Issue: https://github.com/ampproject/amphtml/issues/30052

### CODE REVIEW

**Comment by honeybadgerdontcare** [comment]
There currently isn't a mechanism to define different rule sets for specific extension versions (b/150399834 tracks a feature request to do this). Note though that `data-` attributes are valid for any AMP tag. When they are included in a attribute spec it is to enforce additional constraints on that attribute.

**Comment by krdwan** [comment]
> There currently isn't a mechanism to define different rule sets for specific extension versions (b/150399834 tracks a feature request to do this). Note though that `data-` attributes are valid for any AMP tag. When they are included in a attribute spec it is to enforce additional constraints on that attribute.

Thanks for the context @honeybadgerdontcare !  Makes sense to me.

**Comment by Gregable** [comment]
Validator code looks fine.

**Review by Gregable — APPROVED** [review]

File: extensions/amp-date-countdown/1.0/test/validator-amp-date-countdown.out [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+|        offset-seconds="1521880470"
+|        when-ended="continue"
+|        locale="de"
+|        height="50"
+|        width="360"
+|        template="myTemplate"
+|      >
+|      </amp-date-countdown>
+|    </body>
+|  </html>
```
Comment (caroqliu) [inline extensions/amp-date-countdown/1.0/test/validator-amp-date-countdown.out:294]: nit: new line eof

File: extensions/amp-date-countdown/validator-amp-date-countdown.protoascii [hunk 2]
```diff
@@ -36,6 +36,10 @@ tags: {  # <amp-date-countdown>
     value_casei: "minutes"
     value_casei: "seconds"
   }
+  attrs {
+    name: "count-up"
```
Comment (caroqliu) [inline extensions/amp-date-countdown/validator-amp-date-countdown.protoascii:40]: Please leave a comment that this is noop in 0.1

File: extensions/amp-date-countdown/validator-amp-date-countdown.protoascii [hunk 3]
```diff
@@ -36,6 +36,10 @@ tags: {  # <amp-date-countdown>
...
+    value: ""
+  }
   attrs {
     name: "data-count-up"
```
Comment (caroqliu) [inline extensions/amp-date-countdown/validator-amp-date-countdown.protoascii:44]: Please leave a comment that this is noop in 1.0

**Review by caroqliu — APPROVED** [review]

File: extensions/amp-date-countdown/validator-amp-date-countdown.protoascii (hunk 2 above)
Comment (krdwan) [reply extensions/amp-date-countdown/validator-amp-date-countdown.protoascii:40]: Great, added!

File: extensions/amp-date-countdown/validator-amp-date-countdown.protoascii (hunk 3 above)
Comment (krdwan) [reply extensions/amp-date-countdown/validator-amp-date-countdown.protoascii:44]: Great!  Added!

**Pull request merged**